src/app/add_entry/page.jsx

```diff
@@ -0,0 +1,157 @@
+"use client"
+import Link from 'next/link'
+import { supabase } from '../lib/supabase'
+import { useRouter } from 'next/navigation'
+import { UploadButton } from '../utils/uploadthing'
+import { useState } from 'react'
+
+const weatherOptions = [
+  "sunny",
+  "raining",
+  "cloudy"
+]
+
+function AddEntry() {
+  const router = useRouter()
+
+  const [uploadedFileUrls, setUploadedFileUrls] = useState()
+  const [submitDisabled, setSubmitDisabled] = useState(false)
+
+  const handleSubmit = async (e) => {
+    // Prevent page from refreshing on Submit click
+    e.preventDefault()
```

Comment:
love comments like these that explain that this line is here for a specific but non-obvious reason. Good developer 👍